src/ort_genai_c.h

```diff
@@ -50,6 +50,7 @@ typedef enum OgaElementType {
 typedef struct OgaResult OgaResult;
 typedef struct OgaGeneratorParams OgaGeneratorParams;
 typedef struct OgaGenerator OgaGenerator;
+typedef struct OgaRuntimeSettings OgaRuntimeSettings;
```

Comment:
add the struct for OgaRuntimeSettings ?

We eventually need a way of selecting the EP via API if we want to use the same model for say webgpu and dml. Guess we can use OgaRuntimeSettings  for that too.

Reply:
There will be a struct RuntimeSettings, but in C-API OgaRuntimeSettings is just an opaque type.